faq.html

```diff
@@ -87,7 +87,7 @@ <h4 class="panel-title">
               <div class="panel-body">
                 <p>Open the drawer by swiping from the left or tapping the <i class="fa fa-bars"></i> burger icon on the top left corner. Go to <i class="fa fa-plus"></i> <b>Add Podcast</b>.<br />Now, you can either 
                   search the iTunes catalog, browser gpodder.net or add a podcast by URL.</p>
-                  <p>AntennaPod can also open various protocols from the browser.</p>
+                  <p>You can also click RSS links (and other protocols) from the browser and select AntennaPod to open them.</p>
```

Comment:
"click... and open" seems a bit verbose.
Maybe "You can also open various file formats like RSS or audio with AntennaPod" (actually, this is not restricted to the browser)?